functions/supplier/mail-supplier/README.adoc

```diff
@@ -0,0 +1,34 @@
+# Mail Supplier
+
+This module provides a File supplier that can be reused and composed in other applications.
+The `Supplier` uses the mail IMAP and POP3 support from Spring Integration.
+`MailSupplier` is implemented as a `java.util.function.Supplier`.
```

Comment:
I don't think that we really have a `MailSupplier` class.
Can we somehow change this text to not mislead end-users?